dockwidget.py

```diff
@@ -10,14 +10,14 @@
     os.path.dirname(__file__), 'dockwidget.ui'))
 
 
-class Ui_DockWidget(QtGui.QDockWidget, FORM_CLASS):
+class LizardViewer_DockWidget(QtGui.QDockWidget, FORM_CLASS):
```

Comment:
https://www.python.org/dev/peps/pep-0008/#class-names

Underscores are not necessary.